README.md

```diff
@@ -18,6 +18,73 @@ and then see the results on [localhost:8080/apps/websight/index.html/content::sp
 
 For more details see our [Authoring Quick Start Guide](https://www.websight.io/docs/quick-start/).
 
+
+# Websight Technologies
+
+Find out what technologies Dynamic Solutions team has used to design, build and launch WebSight.
+
+<table>
+     <style>
+     table {
+        border-collapse: collapse;
+     }
+     table, th, td {
+        border: 0px;
+     }
+     </style>
+     <tr>
+     <thead>
+        <tr>
+            <th colspan="2" style='width: 320px'><center>CMS</center></th>
+            <th style='width: 160px'><center>Runtime</center></th>
+            <th colspan="2" style='width: 320px'><center>Supportive</center></th>
+            <th style='width: 160px'><center>Design</center></th>
+        </tr>
+    </thead>
+    <tbody>
+            <td width="100" height="70"><a href="https://www.java.com"><img src="/assets/technologies/java-logo.png" alt="Java" style="width:80px;" align="right"/></a></td>
+            <td width="100" height="70"><a href="https://felix.apache.org"><img src="/assets/technologies/osgi-logo.png" alt="OSGi" style="width:90px;" align="left" /></a></td>
+            <td width="100" height="70"><center><a href="https://aws.amazon.com"><img src="/assets/technologies/aws-logo.png" alt="AWS" style="width:60px;" align="center" /></a></center></td>
```

Comment:
`width`, `height`, `align` attributes and `<center>` tag are deprecated. Maybe we could change them with style="width: 100px; height: 70px; text-align: center"?